l4ping/l4ping.TESTGEN.ps1

```diff
@@ -0,0 +1,143 @@
+#===============================================
+# Script Input Parameters Enforcement
+#===============================================
+Param(
+    [parameter(Mandatory=$false)] [string] $Config = "Default",
```

Comment:
nit: single quotes

Reply:
Copied this from the way the other testgens are working. Would you like me to fix all of them?

Reply:
It's a nit, go ahead and leave as is